packages/react-scripts/per-lang-loader/index.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
```

Comment:
we should leave a comment that this was adapted from alienfast code with a link to it